### ISSUE TITLE
List: assistive technologies, list item does not have ul or ol element

### ISSUE THREAD

**Issue by tkeiner** [SAP/ui5-webcomponents#10359]
### Bug Description

For Lists to be read correctly from assistive technologies like screen readers a certain structure is required.
According to the [Fiori ACC guidelines](https://wiki.one.int.sap/wiki/display/Acc/ACC-264+Details#ACC264Details-conformity) the ui5-list is currently not following the guidelines. When using a tool like the "Access Assisstent" the following issue is found:

_This LI does not have a ul element (without an ARIA-assigned role); ol element (without an ARIA-assigned role); an element with a role set to the value: list as a parent; or a ul element (without an ARIA-assigned role), ol element (without an ARIA-assigned role) or element with a role set to the value 'list' with an aria-owns attribute set to the ID of the element in the same DOM._

![image](https://github.com/user-attachments/assets/b6e38ba0-20e0-4120-99cb-f4ad3009a988)

The issue cant be resolved by users because the li element is controlled by the web components.

### Affected Component

ui5-list

### Expected Behaviour

List should have correct structure to allow content to be read by assistive technology.

### Isolated Example

https://sap.github.io/ui5-webcomponents/components/List/#basic-sample

### Priority

Medium

### UI5 Web Components Version

2.5.0

### Browser

Chrome

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by plamenivanov91** [comment]
Hello @SAP/ui5-webcomponents-topic-p,

Could you please take over?

Regards,
Plamen Ivanov

**Comment by yanaminkova** [comment]
Hello @tkeiner,

Thank you for bringing this issue to our attention. We’re already aware of it and have taken steps to address it.

The component’s DOM structure and ARIA design have been updated, and the updated version is currently in the implementation phase.

Internal Reference: BGSOFUIPIRIN-6285

Kind regatrds,
Yana

**Comment by tkeiner** [comment]
Hi @yanaminkova ,

could you please give me some information when the new DOM Structure for the List will be released?
I saw in the Internal Reference that the item is already completed, however when I checked in v2.8.0 and v2.9.0-rc.0 the DOM structure is not yet updated.

Kind regards,
Tobias

**Comment by tkeiner** [comment]
Hi @yanaminkova,
I just tested again in 2.9.0-rc.2 and still found the issue. Would you mind to reopen the issue?
Kind regards, Tobias
